_episodes/11-software-project.md

```diff
@@ -65,6 +65,7 @@ anything you like, but it may be easier for future group exercises if everyone u
 ![View of the own copy of the software template repository in GitHub](../fig/own-template-repository.png){: .image-with-shadow width="800px" }
 
 > ## Exercise: Obtain the Software Project Locally
+> Make a directory called python-intermediate-inflammation, cd into this folder, run the git init command, then use the clone command.
```

Comment:
Don't think this change is needed.  While you can make the folder and init it in advance of the clone, you don't have to.  If you run git clone it will make a new folder with the name from the clone command, initialize it, setup the remotes, and then pull down the files and history.